### PR TITLE
Add archive first match to 'Manage'

### PR DESCRIPTION
**Pull request by fernandog**
I'm not sure about this. Should I reconfigure columns count or something?
Last time I added a column here the "delete" stopped worked because the column index changed.
Can't remember how to change that? Maybe in JS?

@miigotu @OmgImAlexis

### CODE REVIEW

**Comment by miigotu** [comment]
idk, need to wait @OmgImAlexis

**Comment by fernandog** [comment]
thanks!

Im trying to adjust quality pill to avoid wrap:

1080p WEB
-DL

added ` <td align="center" width="xx">${renderQualityPill(curShow.quality, showTitle=True)}</td>`

but didn't worked

**Comment by miigotu** [comment]
That must be a separate PR, but the column widths are set in css afaik, ask @MGaetan89 about css.

**Comment by fernandog** [comment]
ok. thanks!

**Comment by OmgImAlexis** [comment]
@fernandog you'll need to go through the JS and change everything after that column over by one, for example if that column is now at index 4 find everything with 4 and above and add 1 to it. That'll fix any issues. It should be relatively easy as it's only on one page so just open the `/js/new/manage.js` and also any js that're in the `manage.mako` scripts section.

**Comment by fernandog** [comment]
@OmgImAlexis can't we use column name instead of index ?!

**Comment by OmgImAlexis** [comment]
NO! Index is good for now, I'm planning on changing it all over the data attributes soon.

**Comment by fernandog** [comment]
@OmgImAlexis can you verify? changed manage.js

didn't found anything to change massUpdate.js

File: gui/slick/js/new/manage.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ $(document).ready(function(){
             6: function(node) { return $(node).find("img").attr("alt"); },
             7: function(node) { return $(node).find("img").attr("alt"); },
             8: function(node) { return $(node).find("img").attr("alt"); },
+            9: function(node) { return $(node).find("img").attr("alt"); },
```
Comment (OmgImAlexis) [inline gui/slick/js/new/manage.js]: Is this because you actually changed it for 7 but they're all the same?

Comment (fernandog) [inline gui/slick/js/new/manage.js]: as I added another column that has a img of YES/NO

File: gui/slick/js/new/manage.js [hunk 2]
```diff
@@ -40,17 +41,18 @@ $(document).ready(function(){
...
+           11: { sorter: 'status'},
            12: { sorter: false},
            13: { sorter: false},
            14: { sorter: false},
            15: { sorter: false},
            16: { sorter: false},
-           17: { sorter: false}
+           17: { sorter: false},
+           18: { sorter: false}
         }
```
Comment (fernandog) [inline gui/slick/js/new/manage.js]: I think there's extra here (before my PR)
7 lines with : { sorter: false},

but  6 action columns
```
            <th width="1%">Update<br/><input type="checkbox" class="bulkCheck" id="updateCheck" /></th>
            <th width="1%">Rescan<br/><input type="checkbox" class="bulkCheck" id="refreshCheck" /></th>
            <th width="1%">Rename<br/><input type="checkbox" class="bulkCheck" id="renameCheck" /></th>
        % if sickbeard.USE_SUBTITLES:
            <th width="1%">Search Subtitle<br/><input type="checkbox" class="bulkCheck" id="subtitleCheck" /></th>
        % endif
            <!-- <th>Force Metadata Regen <input type="checkbox" class="bulkCheck" id="metadataCheck" /></th>//-->
            <th width="1%">Delete<br/><input type="checkbox" class="bulkCheck" id="deleteCheck" /></th>
            <th width="1%">Remove<br/><input type="checkbox" class="bulkCheck" id="removeCheck" /></th>
```

Comment (OmgImAlexis) [inline gui/slick/js/new/manage.js]: Yeah I think you may be right about that.

**Comment by OmgImAlexis** [comment]
@fernandog that looks correct now. As long as there's no errors showing up in your console in Chrome you're good to go.

File: gui/slick/views/manage.mako [hunk 3]
```diff
@@ -47,7 +48,7 @@
     <tfoot>
         <tr>
             <td rowspan="1" colspan="2" class="align-center alt"><input class="btn pull-left" type="button" value="Edit Selected" id="submitMassEdit" /></td>
-            <td rowspan="1" colspan="${(14, 15)[bool(sickbeard.USE_SUBTITLES)]}" class="align-right alt"><input class="btn pull-right" type="button" value="Submit" id="submitMassUpdate" /></td>
+            <td rowspan="1" colspan="${(15, 16)[bool(sickbeard.USE_SUBTITLES)]}" class="align-right alt"><input class="btn pull-right" type="button" value="Submit" id="submitMassUpdate" /></td>
```
Comment (fernandog) [inline gui/slick/views/manage.mako]: @OmgImAlexis had to change this too. the black bar in the bottom

**Comment by fernandog** [comment]
Did a mass "update shows" and it worked.

File: gui/slick/views/manage.mako (hunk 3 above)
Comment (OmgImAlexis) [inline gui/slick/views/manage.mako]: Ah, almost missed that. I really need to work on changing it all over to named vars using the data attribute.

**Comment by miigotu** [comment]
@fernandog you aren't supposed to tag your own PR tested unless it was someone else who did the testing. The point is for fresh eyes to check/test/review it in case you are overlooking something. Ofc you tested it if you did the work.

**Comment by fernandog** [comment]
@miigotu should I wait some one test then?

**Comment by miigotu** [comment]
Not if you think it isnt going to cause any issues, but there is no reason to tag it then

**Pull request merged**